src/js/view/view.js

```diff
@@ -255,30 +260,41 @@ define([
                 if (!_lastWidth || !_lastHeight) {
                     _responsiveListener();
                 }
+                _model.set('inDom', inDOM);
+                // Fire resize 0,0 if the player element is not in the DOM
+                // This allows setup to complete even if element was removed from DOM
+                if (!inDOM) {
+                    _resized(containerWidth, containerHeight);
```

Comment:
Yes. `utils.bounds` returns 0 for all dimensions when the body does not contain the element parameter. If the element is in the browser `getBoundingClientRect` will force a repaint and dimensions will be returned, except for maybe IE which will return 0 until after a reflow / repaint.